### PR TITLE
Content about Accessibility

### PR DESCRIPTION
**Pull request by Yemaneberhan-Lemma**
Content about accessibility obtained from [Confluence](https://axelerant.atlassian.net/wiki/spaces/AH/pages/4040824379/Accessibility).

### CODE REVIEW

File: content/docs/how-we-work/accessibility.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: Accessibility
+weight: 9
+description: We implement 508 and WCAG compliant websites so that people with all types of disabilities, whether it be physical, mental, or visual impairment, have access to our sites.
+---
+
+# Accessibility
+
+We implement 508 and WCAG compliant websites so that people with all types of disabilities, whether it be physical, mental, or visual impairment, have access to our sites.
```
Comment (hussainweb) [inline content/docs/how-we-work/accessibility.md:9]: > whether it be physical, mental, or visual impairment

Is this official verbiage? I don't know how WCAG addresses mental impairment. Further, isn't visual impairment physical in nature?

File: content/docs/how-we-work/accessibility.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+- Form elements should have labels.
+- Images should have alt text.
+- Decorative images or images with no content use should either have a null alt tag (alt = "") or rendered as background images.
+- Form buttons should have descriptive values.
+- Color should not be used as the sole method of conveying content or distinguishing visual elements.
+- Color alone is not used to distinguish links from surrounding text unless the luminance contrast between the link and the surrounding text is at least 3:1 and an additional differentiation (e.g., it becomes underlined) is provided when the link is hovered over or receives focus.
+- The page should be readable and functional when the text size is doubled. We need to use rem font-sizing to make the text scales as expected when the web-page is zoomed-in. Also provides reliable text-resizing in smaller browser widths.
+- Need to have adequate line spacing (at least 1/2 the height of the text) and paragraph spacing (1.5 times line spacing).
+
+## [Making Accessible Links](https://www.sitepoint.com/15-rules-making-accessible-links/)
```
Comment (hussainweb) [inline content/docs/how-we-work/accessibility.md:37]: I don't know if Sitepoint is a good reference for us. I believe we have a lot of other references we can link to.  @vishakha1010, can you help?

**Review by hussainweb — CHANGES_REQUESTED** [review]

File: content/docs/how-we-work/accessibility.md (hunk 1 above)
Comment (hussainweb) [reply content/docs/how-we-work/accessibility.md:9]: Let's use the term [neurodivergence](https://wcag.com/blog/digital-accessibility-and-neurodiversity/) instead.

Comment (Yemaneberhan-Lemma) [reply content/docs/how-we-work/accessibility.md:9]: https://axelerant.atlassian.net/wiki/spaces/AH/pages/4040824379/Accessibility

Comment (Yemaneberhan-Lemma) [reply content/docs/how-we-work/accessibility.md:9]: @hussainweb, made as per comments and naming changes as per WCAG.

File: content/docs/how-we-work/accessibility.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Accessibility
 weight: 9
-description: We implement 508 and WCAG compliant websites so that people with all types of disabilities, whether it be physical, mental, or visual impairment, have access to our sites.
+description: We implement 508 and WCAG compliant websites so that people with all types of disabilities, whether it be cognitive impairment, learning disability, physical impairment, sensory impairment, or visual impairment, have access to our sites.
```
Comment (hussainweb) [inline content/docs/how-we-work/accessibility.md:4]: This is better. Please change this in the paragraph below as well.

**Review by hussainweb — CHANGES_REQUESTED** [review]

**Review by hussainweb — APPROVED** [review]
I think you meant to introduce paragraphs but Markdown syntax will put all of them in a same paragraph. I'll merge for now but feel free to change it later.

Also, I think the hyphenated line in table is unnecessarily long. Again, not a stopper for the PR.

**Pull request merged**